core/runtime.rs

```diff
@@ -2433,16 +2433,21 @@ impl JsRuntime {
       let mut args: SmallVec<[v8::Local<v8::Value>; 32]> =
         SmallVec::with_capacity(32);
 
-      while let Poll::Ready(Some(item)) =
-        context_state.pending_ops.poll_next_unpin(cx)
-      {
-        let (promise_id, op_id, mut resp) = item;
+      loop {
+        let mut join_fut = context_state.pending_ops.join_next();
```

Comment:
Ugh. I wonder if we can convince tokio to make `poll_join_next` public. We wouldn't need to pin this block.

Reply:
Opened https://github.com/tokio-rs/tokio/issues/5769